crowd_anki/representation/deck.py

```diff
@@ -75,8 +79,12 @@ def from_collection(cls, collection, name, deck_metadata=None, is_child=False):
                            if Deck.DECK_NAME_DELIMITER
                            not in child_name[len(name) + len(Deck.DECK_NAME_DELIMITER):]]
 
+        # Remove filtered decks from list
+        direct_children_filtered = [child_name for child_name in direct_children
```

Comment:
As this function is recursive, I think the better approach here would be to return None from the calls for filtered decks and filter that on higher level of recursion.